test/testOnly/controllers/PortalStubControllerSpec.scala

```diff
@@ -22,6 +22,7 @@ import controllers.{AuthorisedController, ControllerBaseSpec}
 import play.api.http.Status
```

Comment:
The tests in this spec are failing. I think this is because the `enrolmentsAuthService` used in your controller is using the mock `AuthConnector` defined in `ControllerBaseSpec`, but there is another mock `AuthConnector` defined in this file (not being used). So the mock setup on line 39 is being applied to the unused mock on line 37. I think if you just remove `val mockAuthConnector` from this file then you should be good. Could you also remove the unused imports in here.